context-free.md

```diff
@@ -1,10 +1,10 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+People who want to generate art/images.
```

Comment:
What kind of people? Do they have a science/computer science background or inclination? Why or why not?